src/apps/Plotter.jsx

```diff
@@ -0,0 +1,114 @@
+import functionPlot from "function-plot";
+import { useState } from "react";
+import styled from "styled-components";
+
+//------------Styled Components-------------
+
+const Form = styled.form`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+`;
+
+const Input = styled.input`
+  margin: 2px;
+`;
+
+const H3 = styled.h3`
+  color: red;
+`;
+
+//=========================================
+
+let contentsBounds = document.body.getBoundingClientRect();
+let width = 1500;
+let height = 500;
+let ratio = contentsBounds.width / width;
+width *= ratio;
+height *= ratio;
```

Comment:
Some numbers are being cropped on the top right of the graph after the first plot. 
![image](https://user-images.githubusercontent.com/60931421/135667956-891779d4-a163-496a-a3e2-6025b7c74cd3.png)

Reply:
Made some changes in the width/height ratio and the results were satisfactory with normal display size(screens with width upto 1920 px). For larger screens the graph would be some smaller in size wrt the screen but the co-ordinates at the top-right corner would be visible.

Reply:
Looks better, I like that it doesn't take the whole width of the screen now.